tasks/Task.cpp

```diff
@@ -39,12 +61,10 @@ Task::Task(std::string const& name)
 
 Task::~Task()
 {
+    SetLivoxLidarWorkMode(handle, kLivoxLidarWakeUp, nullptr, nullptr);
+    LivoxLidarSdkUninit();
```

Comment:
You shouldn't be doing this here. The task contexts have a different lifecycle than "plain" C++ objects, where you should pair configure with cleanup and start with stop. Only destroy in here what you have constructed in the constructor.